Microsoft.AVS.Management/Microsoft.AVS.Management.psm1

```diff
@@ -2665,17 +2665,87 @@ function Set-CustomDRS {
         $Clusters += Get-Cluster -Name $cluster_each
     }
 
-    foreach ($cluster in $clusters)
-    {
-        try
-        {
+    foreach ($cluster in $clusters) {
+        try {
             $_this = Get-View -Id $cluster.Id
             $_this.ReconfigureComputeResource_Task($spec, $modify)
             Write-Host "Successfully set DRS for cluster $($cluster.Name)."
         }
-        catch
-        {
+        catch {
             Write-Error "Failed to set DRS for cluster $($cluster.Name)."
         }
     }
 }
+
+Function Set-AVSVSANClusterUNMAPTRIM {
+    <#
+    .DESCRIPTION
+        This function enables vSAN UNMAP/TRIM on the cluster defined by the -Name parameter.  
+        Once enabled, supported Guest OS VM's must be powered off and powered back on.  A reboot will not suffice.
+        See url for more information: https://core.vmware.com/resource/vsan-space-efficiency-technologies#sec19560-sub5
+    .PARAMETER Name
+        Name of Clusters as defined in vCenter.  Valid values are blank or a comma separated list of cluster names.
```

Comment:
Do multi-line descriptions show up correctly to the user? I kinda suspect you may have to constrain the param descriptions to one liner.

Reply:
That's not how user sees it though, they never execute it directly. But that's fine, as long as our metadata ingestion is happy with it.